pkg/plugins/authentication/localjwt/authenticator.go

```
@@ -46,67 +53,91 @@ func New(h kore.Interface, config Config) (identity.Plugin, error) {
 	}
 	log.Info("initializing the jwt authentication plugin")
 
-	return &authImpl{Interface: h, config: config}, nil
+	var key interface{}
+
+	if config.PublicKey != "" {
+		pubkey, err := base64.StdEncoding.DecodeString(config.PublicKey)
+		if err != nil {
+			return nil, fmt.Errorf("Unable to parse public key from config: %v", err)
+		}
+
+		key, err = x509.ParsePKIXPublicKey([]byte(pubkey))
+		if err != nil {
+			return nil, err
+		}
+	}
+	if config.PublicKeyPath != "" {
+		content, err := ioutil.ReadFile(config.PublicKeyPath)
+		if err != nil {
+			return nil, err
+		}
+		key, err = jwt.ParseRSAPublicKeyFromPEM(content)
+		if err != nil {
+			return nil, err
+		}
+	}
```

should we throw an error if BOTH PublicKey and PublicKeyPath are set?